permission_handler_platform_interface/lib/src/permission_status.dart

```diff
@@ -98,10 +109,15 @@ extension FuturePermissionStatusGetters on Future<PermissionStatus> {
   /// *Only supported on iOS.*
   Future<bool> get isRestricted async => (await this).isRestricted;
 
+  /// *On Android:*
   /// If the user denied access to the requested feature and selected to never
-  /// again show a request for this permission. The user may still change the
-  /// permission status in the settings.
-  /// *Only supported on Android.*
+  /// again show a request for this permission,(pre API 30) or the user denied
+  /// permissions for a second time (API 30 and higher).
+  /// The user may still change the permission status in the settings.
+  ///
+  /// *On iOS:*
+  /// If the user has denied acces to the requested feature.
+  /// The user may still change the permission status in the settings
```

Comment:
Should remove the comma:
```suggestion
  /// again show a request for this permission (pre API 30) or the user denied
  /// permissions for a second time (API 30 and higher).
  /// The user may still change the permission status in the settings.
  ///
  /// *On iOS:*
  /// If the user has denied acces to the requested feature.
  /// The user may still change the permission status in the settings
```

Reply:
This suggestion is still open.